composer.json

```diff
@@ -52,24 +52,27 @@
             "bash admin/setup.sh"
         ],
         "analyze": [
+            "Composer\\Config::disableProcessTimeout",
             "phpstan analyze",
-            "psalm"
+            "psalm",
+            "rector process --dry-run"
         ],
+        "sa": "@analyze",
         "ci": [
             "Composer\\Config::disableProcessTimeout",
+            "@cs",
             "@deduplicate",
-            "@analyze",
-            "@test",
             "@inspect",
-            "rector process",
-            "@style"
+            "@analyze",
+            "@test"
         ],
-        "deduplicate": "phpcpd app/ src/",
+        "cs": "php-cs-fixer fix --ansi --verbose --dry-run --diff",
+        "cs-fix": "php-cs-fixer fix --ansi --verbose --diff",
```

Comment:
Was dropping `--using-cache=no` deliberate? I frequently run into issues when switching between Git branches and prefer not to rely on the cache in smaller projects.

Reply:
The code comes from devkit:
https://github.com/codeigniter4/devkit/blob/ddc75f3c36bb60f117b4c7a0817b37115694c6f5/composer.json#L67-L69
and used in Shield and the main repository.

I have never had any trouble and I prefer cash on because it is faster.

Do you want cache off in this repository?

Reply:
No that's fine! Just checking where the change came from. I think default behavior is preferred, and cache should be helpful most of the time.